### PR TITLE
[feed] Embedded music player for each song

### PR DESCRIPTION
**Pull request by hannahgz**
Added embedding for each post in the feed so that the user can directly listen to the song in the application without navigating to another website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,31 +69,30 @@ _One common template for creating a user story is the Who, What, Why template: A
 
 #### Posts
 
-| Property         | Type   | Description                                      |
-| ---------------- | ------ | ------------------------------------------------ |
-| objectId         | String | unique id for the user post (default field)      |
-| createdAt        | Date   | (default auto generated field)                   |
```
Comment (mohan-upadhyay) [inline README.md:75]: Rather than having an generic name like object id, you can rename this column to postId

File: README.md [hunk 2]
```diff
@@ -69,31 +69,30 @@ _One common template for creating a user story is the Who, What, Why template: A
...
-| selectedSongId | String  | unique song id from Spotify                 |
-| postId         | Pointer | pointer to unique id for the user post      |
+| Property       | Type    | Description                                  |
+| -------------- | ------- | -------------------------------------------- |
+| objectId       | String  | unique id for the user post (default field)  |
+| createdAt      | Date    | (default auto generated field)               |
+| updatedAt      | Date    | (default auto generated field)               |
+| comment        | String  | user inputted comment for each post          |
+| selectedSongId | String  | unique song id from Spotify                  |
+| postId         | Pointer | pointer to unique id for the user post       |
```
Comment (mohan-upadhyay) [inline README.md:94]: What is this postId? Is this the objectId in the post table?

File: README.md (hunk 1 above)
Comment (hannahgz) [reply README.md:75]: Unfortunately through Parse it automatically creates objectId as a default field and I don't think I can change it.

File: README.md (hunk 2 above)
Comment (hannahgz) [reply README.md:94]: Yes this postId is the objectId in the post table. Would a name like postObjectId maybe be better?

Comment (mohan-upadhyay) [reply README.md:94]: You can rename the column to postId. That way the object/entity this column is referring to would be more clear.

File: README.md [hunk 3]
```diff
@@ -69,31 +69,30 @@ _One common template for creating a user story is the Who, What, Why template: A
...
-| comment        | String  | user inputted comment for each post         |
-| selectedSongId | String  | unique song id from Spotify                 |
-| postId         | Pointer | pointer to unique id for the user post      |
+| Property       | Type    | Description                                  |
+| -------------- | ------- | -------------------------------------------- |
+| objectId       | String  | unique id for the user post (default field)  |
+| createdAt      | Date    | (default auto generated field)               |
+| updatedAt      | Date    | (default auto generated field)               |
+| comment        | String  | user inputted comment for each post          |
+| selectedSongId | String  | unique song id from Spotify                  |
```
Comment (mohan-upadhyay) [inline README.md:93]: you can simply name the column as songId.

File: README.md [hunk 4]
```diff
@@ -69,31 +69,30 @@ _One common template for creating a user story is the Who, What, Why template: A
...
+| postId         | Pointer | pointer to unique id for the user post       |
+| userObjectId   | String  | object id for user who commented on the post |
```
Comment (mohan-upadhyay) [inline README.md:95]: Could rename this to userId to keep the naming simple

File: README.md [hunk 5]
```diff
@@ -104,29 +103,32 @@ _One common template for creating a user story is the Who, What, Why template: A
 | updatedAt      | Date    | (default auto generated field)              |
 | selectedSongId | String  | unique song id from Spotify                 |
```
Comment (mohan-upadhyay) [inline README.md:104]: songId

File: README.md [hunk 6]
```diff
@@ -104,29 +103,32 @@ _One common template for creating a user story is the Who, What, Why template: A
...
 | postId         | Pointer | pointer to unique id for the user post      |
+| userObjectId   | String  | object id for user who liked the post       |
```
Comment (mohan-upadhyay) [inline README.md:106]: userId

File: project_start/express-api/server.js [hunk 7]
```diff
@@ -160,178 +168,87 @@ app.get('/refresh_token', function(req, res) {
...
-
-    request.get(options, function(error, response, body) {
-      res.status(201).json({body})
-    });
-
-  } catch(err) {
-    next(err)
-  }
-})
+app.use('/post', Post)
```
Comment (mohan-upadhyay) [inline project_start/express-api/server.js:171]: great work on api naming convention

File: project_start/public/index.html [hunk 8]
```diff
@@ -43,5 +43,29 @@
     <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/popper.js@1.12.9/dist/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+    <!-- <script src="https://open.spotify.com/embed-podcast/iframe-api/v1" async></script>
+    <script>
```
Comment (mohan-upadhyay) [inline project_start/public/index.html:47]: remove commented code before merging

**Pull request merged**